packages/amplify-graphql-relational-transformer/src/has-many/has-many-directive-ddb-references-transformer.ts

```diff
@@ -43,5 +43,6 @@ export class HasManyDirectiveDDBReferencesTransformer implements DataSourceBased
   validate = (context: TransformerContextProvider, config: HasManyDirectiveConfiguration): void => {
     ensureReferencesArray(config);
     config.referenceNodes = getReferencesNodes(config, context);
+    // TODO: validate bidirectionality
```

Comment:
do you have these added to the task backlog since these can get lost in the void easily.

Reply:
I'm tracking this in the [feature branch PR](https://github.com/aws-amplify/amplify-category-api/pull/2370)
>  Add validation for relational bi-directionality of DDB references